docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -124,17 +132,20 @@ Each policy includes a rollover and delete definition:
 
 |===
 
-The APM index lifecycle policies can be viewed in {kib}.
+// Is this right?
+// Does where they go in the UI change?
+The APM data stream lifecycle policies can be viewed in {kib}.
```

Comment:
Good point, this will change too. With the migration to DSL, the default lifecycle settings could be viewed at:

Index Management -> Component Templates -> <search_for_apm and look for templates with @lifecycle suffix>